user-impl/src/main/java/Module.java

```diff
@@ -1,12 +1,13 @@
-package com.example.auction.user.impl;
-
 import com.example.auction.user.api.UserService;
+import com.example.auction.user.impl.UserRepository;
+import com.example.auction.user.impl.UserServiceImpl;
 import com.google.inject.AbstractModule;
 import com.lightbend.lagom.javadsl.server.ServiceGuiceSupport;
 
-public class UserModule extends AbstractModule implements ServiceGuiceSupport {
+public class Module extends AbstractModule implements ServiceGuiceSupport {
```

Comment:
I could finally detect the origin of the broken build. I don't have yet full understand how does it happens, but it's related with the order of wiring of Guice modules. 

It seems that when using a non-standard location for the Guice module, the bit that will make the akka cluster to join itself is affected by the bootstrap of cassandra. Probably by the loading of the Cassandra Module.

What I have observed is that sometimes (1 over 3 times), the cluster joining (`JoinClusterImpl.join`) is only called after some Cassandra setup. 

```
2017-07-31 11:17:32,128 [INFO] from akka.cluster.Cluster(akka://ServiceTest_170731111724385) - Cluster Node [akka.tcp://ServiceTest_170731111724385@127.0.0.1:53609] - Starting up...
2017-07-31 11:17:32,214 [INFO] from akka.cluster.Cluster(akka://ServiceTest_170731111724385) - Cluster Node [akka.tcp://ServiceTest_170731111724385@127.0.0.1:53609] - Registered cluster JMX MBean [akka:type=Cluster]
2017-07-31 11:17:32,215 [INFO] from akka.cluster.Cluster(akka://ServiceTest_170731111724385) - Cluster Node [akka.tcp://ServiceTest_170731111724385@127.0.0.1:53609] - Started up successfully
2017-07-31 11:17:32,236 [INFO] from akka.cluster.Cluster(akka://ServiceTest_170731111724385) - Cluster Node [akka.tcp://ServiceTest_170731111724385@127.0.0.1:53609] - No seed-nodes configured, manual cluster join required
...
2017-07-31 11:17:35,577 [DEBUG] from com.datastax.driver.core.ControlConnection - [Control connection] Refreshing schema for servicetest_170731111724385
2017-07-31 11:17:39,781 [INFO] from org.apache.cassandra.auth.CassandraRoleManager - Created default superuser role 'cassandra'

// JoinClusterImpl.join is called here
2017-07-31 11:17:54,137 [INFO] from akka.cluster.Cluster(akka://ServiceTest_170731111724385) - Cluster Node [akka.tcp://ServiceTest_170731111724385@127.0.0.1:53609] - Node [akka.tcp://ServiceTest_170731111724385@127.0.0.1:53609] is JOINING, roles []
2017-07-31 11:17:54,150 [INFO] from akka.cluster.Cluster(akka://ServiceTest_170731111724385) - Cluster Node [akka.tcp://ServiceTest_170731111724385@127.0.0.1:53609] - Leader is moving node [akka.tcp://ServiceTest_170731111724385@127.0.0.1:53609] to [Up]
```

When this happens, the `UserRepositoryTest` fails because it depends on async call to `ClusterStartupTaskActor` in order to get an instance of `CassandraOffsetDao` which is required by the cassandra read side.